ingestion/functions/parsing/mexico/mexico.py

```diff
@@ -0,0 +1,2706 @@
+    "32047": "TEÚL DE GONZÁLEZ ORTEGA",
+    "32048": "TLALTENANGO DE SÁNCHEZ ROMÁN",
+    "32049": "VALPARAÍSO",
+    "32050": "VETAGRANDE",
+    "32051": "VILLA DE COS",
+    "32052": "VILLA GARCÍA",
+    "32053": "VILLA GONZÁLEZ ORTEGA",
+    "32054": "VILLA HIDALGO",
+    "32055": "VILLANUEVA",
+    "32056": "ZACATECAS",
+    "32057": "TRANCOSO",
+    "32058": "SANTA MARÍA DE LA PAZ",
+}
+
+
+def convert_location(state_code: str, municipality_code: str):
+    """
+    Codes beginning with 9 denote missing information
+    """
+    query_list = []
+    if state_code[0] != "9":
+        query_list.append(_STATES[state_code])
+    if municipality_code[0] != "9":
+        query_list.append(_MUNICIPALITIES[state_code + municipality_code])
+    query_string = ", ".join(query_list + ["MEXICO"])
+    return {"query": query_string}
+
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+    """
+    date = datetime.strptime(raw_date, "%Y-%m-%d")
+    return date.strftime("%m/%d/%YZ")
+
+
+def convert_gender(raw_gender: str):
+    if raw_gender == "2":
+        return "Male"
+    if raw_gender == "1":
+        return "Female"
+
+
+def convert_events(date_admitted, hospitalized, icu, date_death):
+    events = [
+        {
+            "name": "confirmed",
+            "dateRange": {
+                "start": convert_date(date_admitted),
+                "end": convert_date(date_admitted),
+            },
+        }
+    ]
+    if hospitalized == "2":
+        events.append(
+            {
+                "name": "hospitalAdmission",
+                "dateRange": {
+                    "start": convert_date(date_admitted),
+                    "end": convert_date(date_admitted),
+                },
+                "value": "Yes",
+            }
+        )
+    if icu == "1":
+        events.append(
+            {
+                "name": "icuAdmission",
+                "dateRange": {
+                    "start": convert_date(date_admitted),
+                    "end": convert_date(date_admitted),
+                },
+                "value": "Yes",
+            }
+        )
+    if date_death != "9999-99-99":
+        events.append(
+            {
+                "name": "outcome",
+                "dateRange": {
+                    "start": convert_date(date_death),
+                    "end": convert_date(date_death),
+                },
+                "value": "Death",
+            }
+        )
+    return events
+
+
+def convert_demographics(gender: str, age: str, nationality: str, national_origin: str):
+    demo = {}
+    if gender:
+        demo["gender"] = convert_gender(gender)
+    if age:
+        demo["ageRange"] = {"start": float(age), "end": float(age)}
+    if nationality == "1":
+        demo["nationalities"] = "Mexican"
+    elif nationality == "2":
+        demo["nationalities"] = national_origin
+    if demo == {}:
+        return None
+    return demo
```

Comment:
If you'd like, can shorten ~equivalently to `return demo or None`

Reply:
Cool!!!!